v7/hack/templates/base.tmpl

```diff
@@ -0,0 +1,38 @@
+## -*- coding: utf-8 -*-
+<%namespace name="base" file="base_helper.tmpl" import="*"/>
+<%namespace name="header" file="base_header.tmpl" import="*"/>
+<%namespace name="footer" file="base_footer.tmpl" import="*"/>
+<%namespace name="annotations" file="annotation_helper.tmpl"/>
+${set_locale(lang)}
+${base.html_headstart()}
+<%block name="extra_head">
+### Leave this block alone.
+</%block>
+${template_hooks['extra_head']()}
+</head>
+% if not HACK_VARIANT:
+    <body class="hack">
+% elif HACK_VARIANT == 'dark':
```

Comment:
You could just use `${HACK_VARIANT}` as a variable (both here and in base_helper).